src/commands/app/add/event.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseCommand = require('../../../BaseCommand')
+const yeoman = require('yeoman-environment')
+const debug = require('debug')('aio-cli-plugin-app:init')
```

Comment:
I don't think we should use debug anymore..
could you switch to aioLogger.debug ?
I would change the label too with the @adobe prefix..
`const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:add:event', { provider: 'debug' })`

Reply:
please do this everywhere :)